### PR TITLE
PM-1583 add envVars to US helm chart

### PR DESCRIPTION
**Pull request by simisimis**

### CODE REVIEW

File: uptime-service-backend/values.yaml [hunk 1]
```diff
@@ -29,6 +29,16 @@ backend:
       path:
   metrics:
     enabled: false
+  envVars: []
```
Comment (Smorci) [inline uptime-service-backend/values.yaml:32]: **suggestion:** we could call this `extraEnvVars` like [bitnami](https://github.com/bitnami/charts/blob/b00508c3817d94635b52fde1042597cb9261b57b/bitnami/redis/values.yaml#L209) to not overshadow the fact that these options do not override the abstracted env vars of the application.
```suggestion
  extraEnvVars: []
```

Comment (simisimis) [reply uptime-service-backend/values.yaml:32]: I agree. extraEnvVars is more descriptive. I was just being consistent with a handful of helm chart that already have envVars

**Review by Smorci — APPROVED** [review]
Good job!

**Review by johnmarcou — APPROVED** [review]
LGTM

**Pull request merged**